Microsoft.DotNet.Interactive/KernelBase.cs

```diff
@@ -84,29 +85,47 @@ command switch
                             context,
                             next),
 
+                        LoadExtensionInNuGetPackage loadCSharpExtension =>
```

Comment:
Is this is C#-specific? If so, it should go into the `CSharpKernel`. This is worth discussing with @TIHan.

Reply:
This is something we need to have a discussion with @KevinRansom about as he has been working on #r nuget for F#. It was mentioned that we could and probably should share so that there isn't a difference in behavior between two implementations of resolving nuget packages for the interactive experience.

I imagine that F# by itself could have its own way of resolving nuget packages, but through an API allow a custom handler.

Reply:
I don't think we need a different way to resolve a NuGet package but might need custom handling once it's resolved, if only for different extension paths within the package.